cmd/web.go

```diff
@@ -168,6 +168,8 @@ func runWeb(ctx *cli.Context) error {
 
 	bindIgnErr := binding.BindIgnErr
 
+	m.Use(user.GetNotificationCount)
```

Comment:
Doesn't this add unnecessary overhead for endpoints like `/api` that will _never_ use it? :)